cmake/modules/unittest.cmake

```diff
@@ -55,6 +55,15 @@ include(${ZEPHYR_BASE}/cmake/kobj.cmake)
 add_dependencies(test_interface ${KOBJ_TYPES_H_TARGET})
 gen_kobj(KOBJ_GEN_DIR)
 
+FILE(TOUCH ${APPLICATION_BINARY_DIR}/zephyr/include/generated/offsets.h)
+FILE(TOUCH ${APPLICATION_BINARY_DIR}/zephyr/include/generated/syscall_list.h)
+FILE(TOUCH ${APPLICATION_BINARY_DIR}/zephyr/include/generated/devicetree_generated.h)
```

Comment:
other CMake function calls are written in lower case, so please adjust to `file(...)` if you continue this approach.

Reply:
How did I miss that? Thanks